src/groupadd.c

```diff
@@ -125,7 +125,7 @@ usage (int status)
 	(void) fputs (_("  -r, --system                  create a system account\n"), usageout);
 	(void) fputs (_("  -R, --root CHROOT_DIR         directory to chroot into\n"), usageout);
 	(void) fputs (_("  -P, --prefix PREFIX_DIR       directory prefix\n"), usageout);
-	(void) fputs (_("  -U, --users USERS             list of user members of this group\n"), usageout);
+	(void) fputs (_("  -U, --users USERS             list of users to assign to the group\n"), usageout);
```

Comment:
Please make the wording consistent with groupmod(8).  We recently improved the wording of the help of groupmod(8).

See:

-  <https://github.com/shadow-maint/shadow/issues/1313>